src/System.IO.Abstractions.Extensions/IDirectoryInfoExtensions.cs

```diff
@@ -199,18 +199,36 @@ public static string[] DiffPaths(this IDirectoryInfo ancestor, IDirectoryInfo ch
         /// </summary>
         /// <param name="source">Source directory</param>
         /// <param name="destination">Destination directory</param>
-        /// <param name="recursive">If true the copy will be recursive and will include subfolders of <paramref name="info"/>. Defaults to false</param>
+        /// <param name="recursive">If true the copy will be recursive and will include subfolders of <paramref name="source"/>. Defaults to false</param>
+        /// <param name="overwrite">If true the copy will overwrite any existing files in <paramref name="destination"/>. Defaults to false</param>
         /// <param name="filesSearchPattern">Search pattern to apply when searching files, defaults to '*'</param>
         /// <param name="directoriesSearchPattern">Search pattern to apply when searching directories, defaults to '*'</param>
         public static void CopyTo(
             this IDirectoryInfo source,
             IDirectoryInfo destination,
             bool recursive = false,
+            bool overwrite = false,
```

Comment:
adding a new parameter in the method like this will brake existing code. can you move as a last parameter.